tests/fixtures/module-unification-addon/tests/dummy/src/resolver.js

```diff
@@ -0,0 +1,13 @@
+import Resolver from 'ember-resolver/resolvers/fallback';
```

Comment:
I suggest we should have this use the glimmer wrapper instead of the fallback resolver:

```js
import Resolver from 'ember-resolver/resolvers/glimmer-wrapper';
```

The glimmer wrapper only implements module unification rules. The fallback is only needed if you have classic addons.

Reply:
I think all apps have classic addons (literally all addons are “classic” right now).

Reply:
The dummy app of an brand new module unification addon should not have an classic addons.

Reply:
Definitely seems ideal, but I highly doubt it is the current state.

Reply:
What is the consensus here?

Reply:
Ok! We had some more discussion. Lets stick with the fallback resolver. We should use the fallback setup as described at https://github.com/rwjblue/ember-module-migrator#running-module-unification-with-fallback-to-classic-app-layout. We want to use this setup for MU apps since there are still classic addons in the default `package.json` and it is reasonable to expect they work out of the box.

I suggest that we should remove the fallback setup before we remove any feature flags here. I've added that step to our blocker list for removing the feature flags.

Reply:
Added!